requirements/system/user-client.tex

```diff
@@ -1,9 +1,291 @@
+      \item Importance: Primary
+      \item Precondition: User must be logged in.
+      \item Post condition (success): A description of the available containers is displayed.
+      \item Post condition (fail):
+      \begin{itemize}
+        \item Networking error.
+        \item An appropriate error message is displayed.
+      \end{itemize}
+      \item Triggering Event: The user requests containers specifications.
+      \item Description:
+      \begin{itemize}
+        \item The user requests containers specifications with the "add command".
+      \end{itemize}
+    \end{itemize}
+
+    \item  UCFR09 Choosing the parameters of the container(preferred OS/environment)
+    \begin{itemize}
+      \item Goal: User chooses preffered container parameters for a specific job.
+      \item Importance: Primary
+      \item Precondition: User must be logged in.
+      \item Post condition (success): An appropriate success message is displayed.
+      \item Post condition (fail):
+      \begin{itemize}
+        \item unavailable specifications.
+        \item An appropriate error message is displayed.
+      \end{itemize}
+      \item Triggering Event: The user chooses containers environment.
+      \item Description:
+      \begin{itemize}
+        \item The user chooses containers environment in the "add command".
+      \end{itemize}
+    \end{itemize}
+
+    \item  UCFR10 Choosing the running mode
+    \begin{itemize}
+      \item Goal: User chooses preffered mode for the application to use.
+      \item Importance: Primary
+      \item Precondition: User must be logged in.
+      \item Post condition (success): An appropriate success message is displayed.
+      \item Post condition (fail):
+      \begin{itemize}
+        \item unavailable mode.
+        \item An appropriate error message is displayed.
+      \end{itemize}
+      \item Triggering Event: The user chooses a specific mode.
+      \item Description:
+      \begin{itemize}
+        \item The user chooses the application mode in the "add command".
+        \item The blocking mode lets the application run until the job is finished or stopped.
+      \end{itemize}
+    \end{itemize}
+
+    \item  UCFR11 Choosing the minimal hardware requirements for a job
+    \begin{itemize}
+      \item Goal: User chooses the minimal hardware required for the job to run.
+      \item Importance: Primary
+      \item Precondition: User must be logged in.
+      \item Post condition (success): An appropriate success message is displayed.
+      \item Post condition (fail):
+      \begin{itemize}
+        \item unavailable requirements.
+        \item An appropriate error message is displayed.
+      \end{itemize}
+      \item Triggering Event: The user chooses the minimal hardware requirements for the job.
+      \item Description:
+      \begin{itemize}
+        \item The user chooses the the minimal hardware requirements in the "add command".
```

Comment:
The user sets the minimum hardware requirements when calling the "add" command of the CLI of the user client.